src/components/pages/CaseTable/CaseTable.js

```diff
@@ -494,29 +497,79 @@ export default function CaseTable(props) {
     });
 
     return (
+      // <Plot
+      //   data={[
+      //     {
+      //       type: 'pie',
+      //       values: [granted, denied, remanded, sustained, terminated],
+      //       labels: [
+      //         'Granted',
+      //         'Denied',
+      //         'Remanded',
+      //         'Sustained',
+      //         'Terminated',
+      //       ],
+      //       textinfo: 'label+percent',
+      //       textposition: 'outside',
+      //       automargin: true,
+      //     },
+      //   ]}
+      //   layout={{
+      //     height: 300,
+      //     width: 300,
+      //     showlegend: false,
+      //     title: 'Decision Rate',
+      //   }}
+      // />
       <Plot
         data={[
           {
-            type: 'pie',
-            values: [granted, denied, remanded, sustained, terminated],
-            labels: [
-              'Granted',
-              'Denied',
-              'Remanded',
-              'Sustained',
-              'Terminated',
-            ],
-            textinfo: 'label+percent',
-            textposition: 'outside',
-            automargin: true,
+            type: 'bar',
+            x: ['Granted', 'Denied', 'Remanded', 'Sustained', 'Terminated'],
+            y: [granted, denied, remanded, sustained, terminated],
           },
         ]}
-        layout={{
-          height: 300,
-          width: 300,
-          showlegend: false,
-          title: 'Decision Rate',
-        }}
+        layout={{ width: 500, height: 300, title: 'Decision Rate' }}
```

Comment:
This code looks so clean, nice job!

Reply:
Thanks for working with DS to get this terminology down!